src/about.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+    <title>Tibeb :About</title>
+  </head>
+  <body>
+    <div class="mobile-menu display-none">
+      <img src="../assets/x-icon.png" alt="x icon" class="x-icon" />
+    </div>
+    <ul class="top-nav">
+      <li class="top-nav__facebook"><i class="fab fa-facebook-square"></i></li>
+      <li class="top-nav__twiter"><i class="fab fa-twitter-square"></i></li>
+      <li class="top-nav__English">English</li>
+      <li class="top-nav__my-page">My Page</li>
+      <li class="top-nav__logout">Logout</li>
+    </ul>
+
+    <nav>
+      <h2 class="nav__link header__nav--logo dn">
+        <a href="index.html">Tibeb <span>Bootcamp</span></a>
+      </h2>
+      <ul class="nav dn">
+        <li class="nav__link"><a href="about.html">About</a></li>
+        <li class="nav__link"><a href="#">Program</a></li>
+        <li class="nav__link"><a href="#">Join</a></li>
+        <li class="nav__link"><a href="#">Sponsor</a></li>
+        <li class="nav__link"><a href="#">News</a></li>
+      </ul>
+    </nav>
+    <header class="header">
+      <a class="hamburger" href="#"
+        ><h2><i class="header__hamurger--menu-icon fas fa-bars"></i></h2
+      ></a>
+
+      <section class="headline headline-about">
```

Comment:
- [ ] Please make sure to add any visual distinction to the page so you can easily know when a section ends and the next one is starting.
_You can do this by slightly changing the background color for each section (i.e. section number one can have background color white, section number two can have background color grey and section number three can have background color white again) this way the user can easily find the difference between sections and you will also stick a bit better to the original design._